src/Conversion/KrnlToAffine/ConvertKrnlToAffine.cpp

```diff
@@ -617,6 +618,7 @@ void ConvertKrnlToAffinePass::runOnOperation() {
 
   MLIRContext *ctx = &getContext();
   OpBuilder builder(ctx);
+  VectorMachineSupport::setGlobalVectorMachineSupport(march, mcpu, "");
```

Comment:
Need to define a global with the current vector machine support; free it at end of pass so that we have no men leaks.